Exercises/if-else-switch/if-else-switch.js

```diff
@@ -4,11 +4,47 @@ const codeToMissingComplexityElement = {
   2: 'special character',
   3: 'cannot be empty'
 };
+let specialCharacterCheck = false;
+const specialCharacters = ['!', '_', '-', '%', '(', ')', '#', '*', '{', '}'];
+let arrayResult = [];
 
 function getMissingComplexityElementCode(password) {
+  for (const char of password) {
+    if (specialCharacters.includes(char)) {
+      specialCharacterCheck = true;
+    }
+  }
 
+  if (password.length < 8) {
+    arrayResult.push(0);
+  }
+  if (password === password.toLowerCase()) {
+    arrayResult.push(1);
+  }
+  if (!specialCharacterCheck) {
+    arrayResult.push(2);
+  }
+  if (password.length === 0) {
+    arrayResult.push(3);
+  }
 }
 
-function getErrorMessage() {
-  return false;
+function getErrorMessage(array) {
+  for (const char of array) {
+    // eslint-disable-next-line default-case
+    switch (char) {
```

Comment:
the idea here, is to get value of key i.e. `length`, `empty`... and use it in switch cases.